devlib/target.py

```diff
@@ -188,6 +190,10 @@ def connect(self, timeout=None):
         self._update_modules('connected')
         if self.platform.big_core and self.load_default_modules:
             self._install_module(get_module('bl'))
+        if self.busybox is not None:
+            # Busybox is used on this target so pass on to connections
+            # Some connections (i.e. gem5) need to wrap some commands
+            self.conn.set_busybox_wrapper(self.busybox)
```

Comment:
Does this need to be set here? 
If so, the android implementation needs to be updated to be consistent.